### PR TITLE
Update ops framework and charm libs

### PR DESCRIPTION
**Pull request by dragomirp**
## Proposal
* [DPE-1160](https://warthogs.atlassian.net/browse/DPE-1160)
* Update ops framework
* Update charm libs
* Copy over check lib and ignore text files from the postgresql K8S charm to ci.yaml

## Release Notes
* Housekeeping

## Testing

[DPE-1160]: https://warthogs.atlassian.net/browse/DPE-1160?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/relations/backend_database.py [hunk 1]
```diff
@@ -293,7 +293,11 @@ def postgres(self) -> PostgreSQL:
             return None
 
         return PostgreSQL(
-            primary_host=endpoint.split(":")[0], user=user, password=password, database=database
+            primary_host=endpoint.split(":")[0],
+            current_host=endpoint.split(":")[0],
```
Comment (dragomirp) [inline src/relations/backend_database.py:297]: Added to the postgresql lib

File: tox.ini [hunk 2]
```diff
@@ -56,6 +56,7 @@ commands =
 description = Run unit tests
 deps =
     pytest
+    pytest-asyncio
```
Comment (dragomirp) [inline tox.ini:59]: Causes warnings in the unit tests

**Review by marceloneppel — APPROVED** [review]

File: lib/charms/operator_libs_linux/v0/apt.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2021 Canonical Ltd.
```
Comment (taurus-forever) [inline lib/charms/operator_libs_linux/v0/apt.py:1]: :-D time-back machine detected!

**Review by taurus-forever — APPROVED** [review]
Superb!

**Pull request merged**